spec/dummy/_config.yml

```diff
@@ -54,6 +54,11 @@ contentful:
           draft: false
     messages:
       id: message
+      links:
+        series_slug:
+          content_type: series
+          field: videos
+          value: slug
```

Comment:
Lets consider refactoring this logic so its simpler to explain, simpler to configure and does some things for you automatically, via convention vs. configuration.